### PR TITLE
boards/pba-d-01-kw2x: changed UART RX pin setting

### PR DESCRIPTION
**Pull request by mfrey**
The current UART RX setting of the phyNode (pba-d-01-kw2x) conflicts with the RGB red led mapping. If you use the current setting the UART works fine but LED blinks red. This PR changes the UART RX setting to Port A1.

The datasheet of the MKW2xD defines UART RX as Port D6 but the pba-d-01-kw2x uses the same GPIO for the RGB red led (the Arduino Uno compatible Connector [interface table](https://github.com/PHYTEC-Messtechnik-GmbH/phynode-riot-examples/wiki/Interfaces) of the phyNode lists D5 has the RGB red led GPIO (but the picture refers to Port D6), but that's probably an  error).

### CODE REVIEW

**Comment by kaspar030** [comment]
Please state why you closed this, for reference.

**Comment by mfrey** [comment]
tl;dr The new UART RX setting only works with a extension board created for a research project. 

The new UART RX setting only works with our extension board for the phyNode. I've thought that I've got it running without the extension board, but @PeterKietzmann couldn't reproduced it and so I did another two hours debugging and couldn't reproduce the intended behavior. Hence, the closed PR. I'm sorry for the noise.
